<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(0, 0, 0);
border-radius: 20px;
}
</string>
  </property>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgb(46, 52, 54);
border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
   color: rgb(0, 230, 255);    }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);    }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);    }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);  }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
   color: rgb(0, 230, 255);    }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
   color: rgb(0, 230, 255);     }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);  }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);    }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255); }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butdot">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="percent">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);
    border-radius: 24px;                   
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divide">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);             
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);  
}</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiply">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);            
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);                 
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);           
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equals">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(57, 67, 78);                
    color: rgb(0, 230, 255);   }

QPushButton:hover {
    background: rgb(46, 52, 54);           /* Hover background color */
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#00e6ff;&quot;&gt;input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="output">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>116</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#00e6ff;&quot;&gt;output&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>271</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 230, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
